core/src/com/unciv/models/metadata/GameParameters.kt

```diff
@@ -23,7 +23,7 @@ class GameParameters { // Default values are the default new game
     var nuclearWeaponsEnabled = true
     var religionEnabled = false
 
-    var victoryTypes: ArrayList<VictoryType> = arrayListOf(VictoryType.Cultural, VictoryType.Domination, VictoryType.Scientific) // By default, all victory types
+    var victoryTypes: ArrayList<VictoryType> = arrayListOf(VictoryType.Cultural, VictoryType.Diplomatic, VictoryType.Domination, VictoryType.Scientific) // By default, all victory types
```

Comment:
I think diplomatic victory shouldn't be by default, at least not until enough early adopters have tested it